bundle/next/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml

```diff
@@ -247,18 +235,24 @@ spec:
     ```
 
     By opening this URL in a web browser, Eclipse Che is ready to use.
+
     ## Defaults
     By default, the operator deploys Eclipse Che with:
     * Bundled PostgreSQL and Keycloak
     * Common PVC strategy
     * Auto-generated passwords
     * TLS mode (secure ingresses)
+    * Communicate between components using internal cluster SVC names
+
     ## Installation Options
     Eclipse Che operator installation options include:
     * Connection to external database and Keycloak
     * Configuration of default passwords and object names
     * PVC strategy (once shared PVC for all workspaces, PVC per workspace, or PVC per volume)
     * Authentication options
+    Use `kubectl edit checluster/eclipse-che -n my-eclipse-che` to update Eclipse Che configuration.
```

Comment:
Maybe move to a separate section?

Reply:
We don't have many sections and it is just one sentence.